test/z_cluster_SUITE.erl

```diff
@@ -240,10 +245,10 @@ roles(Config) ->
                                             reconnect => false}),
 
     %% try to connect the middle of the chain
-    Ret = shackle:call(middle_end, {fetch, [{Topic, [{0, 12, 100}]}]}),
-    ?assert(Ret =:= {error, socket_closed} orelse
-            Ret =:= {error, no_socket} orelse
-            Ret =:= {error, timeout}),
+    %% Ret = shackle:call(middle_end, {fetch, [{Topic, [{0, 12, 100}]}]}),
```

Comment:
this needs some sort of replacement

Reply:
It is no longer valid. I meant to remov eit, not comment it out.